mobile/envoy_build_config/extension_registry.cc

```diff
@@ -178,6 +182,14 @@ void ExtensionRegistry::registerFactories() {
   // This is required for the default upstream local address selector.
   Upstream::forceRegisterDefaultUpstreamLocalAddressSelectorFactory();
 
+  // This is required for load balancers of upstreams.
+  // TODO(wbpcode): maybe not all these are necessary. If some are unnecessary, we should
+  // compile them out.
+  Envoy::Extensions::LoadBalancingPolices::ClusterProvided::forceRegisterFactory();
+  Envoy::Extensions::LoadBalancingPolices::RoundRobin::forceRegisterFactory();
+  Envoy::Extensions::LoadBalancingPolices::LeastRequest::forceRegisterFactory();
+  Envoy::Extensions::LoadBalancingPolices::Random::forceRegisterFactory();
```

Comment:
yeah I don't think we need any of these - would prefer to keep them out unless it breaks tests

Reply:
got it.